platforms/core-execution/hashing/src/main/java/org/gradle/internal/hash/DefaultFileHasher.java

```diff
@@ -15,12 +15,13 @@
  */
 package org.gradle.internal.hash;
 
+import org.gradle.api.UncheckedIOException;
```

Comment:
Use the Gradle version of this class instead of the Java to avoid a `usesFutureStdlib`